ProteoformSuiteGUI/TheoreticalDatabase.cs

```diff
@@ -191,18 +190,18 @@ private void cmbx_DisplayWhichDB_SelectedIndexChanged(object sender, EventArgs e
             }
         }
 
-        public void FillDataBaseTable(string table)
```

Comment:
I deleted it because the same method exists at line 75 so I think with merging different people's commits there was a repeat of the method, so there was an error.